web/src/user_profile.js

```diff
@@ -576,7 +584,10 @@ export function show_edit_bot_info_modal(user_id, $container) {
             contentType: false,
             success() {
                 avatar_widget.clear();
-                hide_user_profile();
+                hide_button_spinner($submit_btn);
+                $submit_btn.prop("disabled", true);
+                original_values = get_current_values($("#bot-edit-form"));
+                $cancel_btn.prop("disabled", false);
```

Comment:
This probably can just use `toggle_submit_button` rather than directly tweaking the buttons?

Reply:
Fixed it.